permission_handler_android/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 12.0.0
+
+* **BREAKING CHANGES:**
+  * Fixes a bug where `Permission.Phone` would always return 'denied' when requesting the permission status.
+  * Fixes a bug where Flutter permissions that require multiple Android permissions would base their status on the status of the first Android permission, as opposed to the result of all relevant Android permissions.
```

Comment:
I am wondering if these are really breaking changes. They seem like bugs where users already have different expectations from how the current version works and these fixes align to the expectations. 

Personally I think we can make this a patch release.